### PR TITLE
test for pricing page

### PR DESCRIPTION
**Pull request by ElTragon**

### CODE REVIEW

**Review by liuliu-dev — APPROVED** [review]
looks good

File: cypress/e2e/publicPaths/render/pricingPage/index.spec.ts [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+import { allDevicesForAppLayout } from "../../../utils/devices";
+import {
+  newClickFlow,
+  newExpectation,
+  newExpectationWithClickFlows,
+  newExpectationWithScrollIntoView,
+  newShouldArgs,
+} from "../../../utils/helpers";
+
+const pageName = "PricingPage";
```
Comment (tbantle22) [inline cypress/e2e/publicPaths/render/pricingPage/index.spec.ts:11]:
```suggestion
const pageName = "Pricing page";
```

File: cypress/e2e/publicPaths/render/pricingPage/index.spec.ts [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+      datacy: "instance-dropdown",
+      currentValue: "M4",
+      optionId: "react-select-4-option-2",
+      valueToClick: "R5B",
+    },
+  ];
+
+  const tests = [
+    newExpectation(
+      "renders Info container",
```
Comment (tbantle22) [inline cypress/e2e/publicPaths/render/pricingPage/index.spec.ts:37]:
```suggestion
      "should find info container",
```

File: cypress/e2e/publicPaths/render/pricingPage/index.spec.ts [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    ...cards.map(card =>
+      newExpectation(
+        `should find ${card} Card`,
+        `[data-cy=${card}-card]`,
+        beVisibleAndContain(card),
+      ),
+    ),
+
+    newExpectationWithScrollIntoView(
+      "renders Table container",
```
Comment (tbantle22) [inline cypress/e2e/publicPaths/render/pricingPage/index.spec.ts:57]:
```suggestion
      "should find Table container",
```

**Review by tbantle22 — APPROVED** [review]
Looks great! Just some small copy consistency things

**Pull request merged**